### DISCUSSION TITLE
Artemis r6.3

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/ArtemisPS3#50] [General]
## Added

- Alternative `Joker Select` plugin
- Code Type 5 support using Joker plugin (See #22 notes)

## Fixed

- Fix online database URL

## Misc

- Update offline cheat database with latest codes

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/ArtemisPS3/releases/tag/r6.3'>Artemis r6.3</a>.</em>

**Comment by Bred9000** [comment]
how do you create cheat for sniper elite 3 ?

**Comment by Danbencaos** [comment]
Hi guys?  All good?  Any predictions for Artemis PS3 CFW 4.90?

**Comment by sunchip123** [comment]
yeah marvel ultimate alliance cheats for high body focus and strike stats cheats please one day

**Comment by bradondc4** [comment]
Necesito una actualización de 2024 porfavor 😭😭

**Comment by bradondc4** [comment]
Actualización por favor 😭
